### PR TITLE
Edit auto focus fix

### PR DESCRIPTION
**Pull request by AlekhyaYalla**

### CODE REVIEW

File: ui/react-components/components/AppointmentEditorCommonFieldsWrapper/AppointmentEditorCommonFieldsWrapper.jsx [hunk 1]
```diff
@@ -115,7 +115,8 @@ const AppointmentEditorCommonFieldsWrapper = props => {
                     <div data-testid="location-search">
                         <LocationSearch value={appointmentDetails.location}
                                         onChange={(optionSelected) => updateAppointmentDetails({location: optionSelected})}
-                                        isDisabled={componentsDisableStatus.location}/>
+                                        isDisabled={componentsDisableStatus.location}
+                                        autoFocus={componentsDisableStatus.speciality && componentsDisableStatus.service && editAutoFocus}/>
```
Comment (bsneha90) [inline ui/react-components/components/AppointmentEditorCommonFieldsWrapper/AppointmentEditorCommonFieldsWrapper.jsx]: what is editAutoFocus??

Comment (AlekhyaYalla) [reply ui/react-components/components/AppointmentEditorCommonFieldsWrapper/AppointmentEditorCommonFieldsWrapper.jsx]: wheather it need autoFocus or not, which will come from editAppointment.jsx

Comment (bsneha90) [reply ui/react-components/components/AppointmentEditorCommonFieldsWrapper/AppointmentEditorCommonFieldsWrapper.jsx]: AppointmentEditorCommonFieldsWrapper does not care if it is for edit or not, so why is it called editAutoFocus?

Comment (AlekhyaYalla) [reply ui/react-components/components/AppointmentEditorCommonFieldsWrapper/AppointmentEditorCommonFieldsWrapper.jsx]: it need to know if its edit or add screen. So we are passing editAutoFocus from editAppointment

**Pull request merged**